packages/samples/stencil/stencil.config.ts

```diff
@@ -36,6 +38,36 @@ export const config: Config = {
       html({
         include: 'src/components/**/*.html',
       }),
+      resolveAtomicPaths(),
     ],
   },
 };
+
+function resolveAtomicPaths() {
```

Comment:
The exports field change was causing issues here as well because moduleResolution could not resolve @coveo/atomic/loader as @coveo/atomic/dist/loader. This function resolve those paths to look for the node_modules directly.

Reply:
hackish as F, but could we instead trick Stencil by including:

```json
"./dist/loader": {
  "types": "./dist/loader.index.d.ts",
  "import": "./dist/loader/index.js",
  "require": "./dist/loader/index.cjs.js"
},
```

in the Atomic package.json? 😅

Reply:
No it is meant to be hackish. Stencil has problems with the "exports" field. I would not change our package to match stencil needs. This is a good example of how to go around that problem imo !